robyn/__init__.py

```diff
@@ -69,6 +73,18 @@ def __init__(
         self.exception_handler: Optional[Callable] = None
         self.authentication_handler: Optional[AuthenticationHandler] = None
 
+    def _handle_dev_mode(self):
+        cli_dev_mode = self.config.dev  # --dev
+        env_dev_mode = os.getenv("ROBYN_DEV_MODE", "False").lower() == "true"  # ROBYN_DEV_MODE=True
+        is_robyn = os.getenv("ROBYN_CLI", False)
+
+        if cli_dev_mode and not is_robyn:
+            raise SystemExit("Dev mode is not supported in the python wrapper. Please use the Robyn CLI. e.g. python3 -m robyn app.py --dev")
+
+        if env_dev_mode and not is_robyn:
+            logger.error("Ignoring ROBYN_DEV_MODE environment variable. Dev mode is not supported in the python wrapper.")
+            raise SystemExit("Dev mode is not supported in the python wrapper. Please use the Robyn CLI. e.g. python3 -m robyn app.py")
```

Comment:
@sansyrox we probably shouldn't raise an exit here; this would require the user to go to the env file & update it, just to make the app run. how about warning them & proceeding?

Reply:
@VishnuSanal , the file should be used locally. The flags should be toggled through a menu. 

I'd want to keep it as is for now. Should there be some concerns, we can allow it to run 😄